docs/src/installation/Raspberry-Pi-32.md

```diff
@@ -0,0 +1,123 @@
+nano ~/.config/systemd/user/spotifyd.service
+systemctl --user daemon-reload
+```
+
+Or in the user diretcory of systemd:
+
+```bash
+sudo nano /etc/systemd/user/spotifyd.service
+```
+
+## Configuring spotifyd
+
+Spotifyd comes pre-configured with defaults that should be working in most cases, but if you want to tweak it further to your needs, have a look at File.md at the [configuration section](../config/) of this book.
+
+## Start the service
+
+As user daemon
+
+```bash
+systemctl --user start spotifyd.service
+```
+
+As system daemon
+
+```bash
+sudo systemctl start spotifyd.service
+```
+
+Now see if you can find it in the normal Spotify client (Devices in right bottom corner). Retry the above steps if you can't find it.
+
+## Starting spotifyd at boot
+
+As system daemon:
+
+```bash
+sudo systemctl enable spotifyd.service
+```
+
+As user daemon is not recommended, it crashes currently with Message: called 'Result::unwrap()' on an 'Err' value: DnsSdError(Os { code: 19, kind: Uncategorized, message: "No such device" }):
+
+```bash
+sudo loginctl enable-linger <username>
+systemctl --user enable spotifyd.service
+```
+
+The first command is required to enable your user to run long-running services. Without it `systemd` would kill the `spotifyd` process as soon as you log out, and only run it when you log in.
+
+Now `spotifyd` is always running on the Pi, so you can use it as a listening device remotely!
+
+## Starting with dbus enabled at boot
+
+If you have dbus enabled, you need to to configure spotifyd to use the system dbus as it can't register services on the session dbus at boot.
+You need to have a config file and configure to use the system dbus, see `Configuring spotifyd`, you can delete everything you don't need to use the standard values. But you need to configure 2 values:
+
+```config
+use_mpris = true
+dbus_type = "system"
+```
+
+In the standard DBus config, no one is allowed to register services on the system dbus, so you need to configure it. Create a configuration file under /usr/share/dbus-1/system.d/, eg. spotifyd-dbus.conf (must end with .conf)
+
+```bash
+sudo nano /usr/share/dbus-1/system.d/spotifyd-dbus.conf
+```
+
+Add the following content:
+
+```content
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+
+<busconfig>
+    <policy user="root">
+        <allow own_prefix="org.mpris.MediaPlayer2.spotifyd"/>
+        <allow send_type="method_call" log="true"/>
+    </policy>
+    <policy context="default">
+    </policy>
+</busconfig>
+```
+
+If you try to run as user, replace root with the user account name.
+
+## Known issues / Logging
+Logging in files is currently not possible for the daemon setup, the daemon crashes without the --no-daemon argument (redirects the output to stdout).
+Without the parameter it tries to output at syslog, but even with syslog installed it crashed.
+To get at least the latest output run
+
```

Comment:
I'm not entirely sure what you mean here. When run without `--no-daemon`, `spotifyd` just logs to the `syslog`. So running `journalctl --follow SYSLOG_IDENTIFIER=spotifyd`, you should be able to see the relevant messages.

In the systemd setup with `--no-daemon`, the output should also appear in `journalctl -u spotifyd.service`.

Reply:
Let's track this further in the issue.